doc/user/content/concepts/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Concepts"
```

Comment:
I didn't promote Concepts from out of Get Started section and be its own top-level section at this time is so that people can flow from the Get Started section to the Ingest data section.   This might be revisited as I track our analytics a bit more.